### PR TITLE
Clarify and standardize directory structure and venv location

### PR DESCRIPTION
**Pull request by encukou**
- Get rid of the `~/pyladies` or `~/naucse-python` that Windows users had to replace with something else throughout the course (#313). Only use it as an example within one lesson.
- Make the learners create numbered directories for lessons, to reduce chaos about where they were saving the files.
- In the first few lessons, give more detailed instructions on saving files.

There's also a few unrelated fixes/clarifications. (I can't help but edit when I read something, and pulling them into separate commits involves too much rebasing. Sorry!)

I've merged this into my "experimental" course, so a render is here: https://naucse.python.cz/2018/pyladies-brno-podzim-exp/sessions/preparation/

### CODE REVIEW

File: lessons/beginners/comparisons/index.md [hunk 1]
```diff
@@ -93,24 +93,23 @@ print(False)
...
-
 Teď oprášíme program na výpočet obvodu a obsahu.
-Do nového souboru, třeba <code><span class="pythondir">~/{{ rootname }}</span>/02/if.py</code>,
-napiš následující program.
-Co se stane, když jako stranu zadáš záporné číslo?
-Dává výstup smysl?
+
+Otevři si v editoru nový soubor.
+V adresáři, kde máš soubory ke kurzům Pythonu, udělej adresář s číslem lekce
+(třeba `02`).
```
Comment (hroncok) [inline lessons/beginners/comparisons/index.md]: Can we get the lesson numerical ID programmatically? Only if it's easy.

File: lessons/beginners/comparisons/index.md [hunk 2]
```diff
@@ -93,24 +93,23 @@ print(False)
...
+
+Do souboru pak napiš následující program:
 
 ```python
 strana = float(input('Zadej stranu čtverce v centimetrech: '))
 print('Obvod čtverce se stranou', strana, 'je', 4 * strana, 'cm')
 print('Obsah čtverce se stranou', strana, 'je', strana * strana, 'cm2')
 ```
 
+Co se stane, když jako stranu zadáš záporné číslo?
```
Comment (hroncok) [inline lessons/beginners/comparisons/index.md]: Tady nějak chybí, že ten soubor máš spustit. A pokud se to hodí, zopakoval bych jak.

File: lessons/beginners/variables/index.md [hunk 3]
```diff
@@ -16,14 +16,8 @@ a obsah jako <var>S</var> = <var>a</var>².
...
 
-{% if var('pyladies') -%}
-{% set rootname = 'pyladies' %}
-{%- else -%}
-{% set rootname = 'naucse-python' %}
-{%- endif %}
-
 Výsledky vypiš pomocí `print()`.
-Program ulož do <code><span class="pythondir">~/{{ rootname }}</span>/02/ctverec.py</code>
+Program ulož do do adresáře pro dnešní lekci `ctverec.py`
```
Comment (hroncok) [inline lessons/beginners/variables/index.md]: Chybí mi tam slovo. Co "jako ctverec.py"?

File: lessons/beginners/comparisons/index.md (hunk 1 above)
Comment (encukou) [reply lessons/beginners/comparisons/index.md]: It's not particularly hard, but a bit lower priority for me. Filed as https://github.com/pyvec/naucse.python.cz/issues/444.

File: lessons/beginners/comparisons/index.md (hunk 2 above)
Comment (encukou) [reply lessons/beginners/comparisons/index.md]: Už je to v téhle lekci potřetí, nechal bych to tady bez detailních instrukcí.

**Pull request merged**